Example/Sources/View Controllers/ChatViewController.swift

```diff
@@ -313,18 +319,19 @@ extension ChatViewController: InputBarAccessoryViewDelegate {
         }
 
         let components = inputBar.inputTextView.components
-        messageInputBar.inputTextView.text = String()
-        messageInputBar.invalidatePlugins()
-
+        inputBar.inputTextView.text = String()
+        inputBar.invalidatePlugins()
         // Send button activity animation
-        messageInputBar.sendButton.startAnimating()
-        messageInputBar.inputTextView.placeholder = "Sending..."
+        inputBar.sendButton.startAnimating()
+        inputBar.inputTextView.placeholder = "Sending..."
+        // Resign first responder for iPad split view
```

Comment:
@ashleyng I was wondering what the purpose behind resigning the first responder is. What was the cause for adding this to the example? I am currently working through an iPad Split View implementation and noticed your comment in the example.

Thanks!

Reply:
I would also like to know this.